tests/components/zwave_js/test_sensor.py

```diff
@@ -131,16 +130,6 @@ async def test_disabled_indcator_sensor(
     assert entity_entry.disabled_by == er.DISABLED_INTEGRATION
 
 
-async def test_disabled_basic_sensor(hass, ge_in_wall_dimmer_switch, integration):
```

Comment:
Can we get a hold of a dump for a basic CC without target value, so we can keep this test?

Reply:
Trying to get one 👍🏾

Reply:
I originally kept the sensor discovery logic in because @kpine thought there may be a use case for it but it appears there isn't one (Al said as much yesterday when we discussed it) so I have removed the sensor discovery logic and related sensor platform code entirely